### PR TITLE
add quick reg tests to CI

### PR DESCRIPTION
**Pull request by royjacobson**

### CODE REVIEW

**Comment by dranikpg** [comment]
What I thought would be cool: after every long regtest run we filter out those which are quick and run only them. This way we don't have to keep track of this marker manually 😛

**Comment by chakaz** [comment]
> What I thought would be cool: after every long regtest run we filter out those which are quick and run only them. This way we don't have to keep track of this marker manually 😛

We can maintain this state in a Dragonfly! :rofl:

File: .github/actions/regression-tests/action.yml [hunk 1]
```diff
@@ -16,7 +16,12 @@ inputs:
   build-folder-name:
     required: true
     type: string
-
+  filter:
+    required: false
+    type: string
+  report-errors:
```
Comment (adiholden) [inline .github/actions/regression-tests/action.yml:22]: @royjacobson I dont see you used it.. did you want to use it for the send notifications condition?

File: .github/actions/regression-tests/action.yml [hunk 2]
```diff
@@ -47,13 +52,12 @@ runs:
...
-        pytest --json-report --json-report-file=rep1_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=true
-        pytest --json-report --json-report-file=rep2_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=false
-
+        pytest -m "${{inputs.filter}}" --json-report --json-report-file=rep1_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=true
+        pytest -m "${{inputs.filter}}" --json-report --json-report-file=rep2_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=false
 
 
     - name: Send notification on failure
-      if: failure()
+      if: failure() && ${{ inputs.run-only-on-ubuntu-latest == 'true'}}
```
Comment (adiholden) [inline .github/actions/regression-tests/action.yml:60]: why only on run-only-on-ubuntu-latest ?

File: .github/actions/regression-tests/action.yml (hunk 1 above)
Comment (royjacobson) [reply .github/actions/regression-tests/action.yml:22]: yeah. now with the 'only on main' condition added it's probably redundant.

File: .github/actions/regression-tests/action.yml (hunk 2 above)
Comment (royjacobson) [reply .github/actions/regression-tests/action.yml:60]: I think there might have a been a problem with the other image, maybe pip/pip3 or something silly like that

**Pull request closed without merging**